submissions/dyworld-history/dyworld-history.md

```diff
@@ -0,0 +1,49 @@
+# DyWorld-Dynamics and it's history <author>Dysoch</author>
+
+Hello everybody, my name is Dysoch. In this article I will write a bit about my history with modding Factorio, why I am doing it and what is coming. So, lets get to the history
```

Comment:
This might need a little more juice to it :) Maybe something like

> Hello everybody, my name is Dysoch, and I'm the creator of all the Dy-Franchise mods. In this article I will write a bit about my many years of history with modding Factorio, and the next exciting release that I have been working on for almost a year. But first, I will explain where it all started.